src/ui/certificatelistwidget.hpp

```diff
@@ -19,13 +17,10 @@ class CertificateListWidget : public QWidget
     CardCertificateAndPinInfo selectedCertificate() const;
     void selectFirstRow();
 
-signals:
+Q_SIGNALS:
```

Comment:
As discussed, I suggest not changing the Qt lowercase macros yet.

Here are my arguments:

- the topic is still widely debated and there is no consensus what is the path forward; people have suggested omitting `emit` altogether, using C++11 function attributes `[[qt::slot]], [[qt::signal]]` etc. (See the thread [here](https://lists.qt-project.org/pipermail/development/2020-February/thread.html#38924), Kai Köhne's idea to get rid of `emit` [here](https://lists.qt-project.org/pipermail/development/2020-February/038883.html) and Lars Knoll's suggestion to use function attributes [here](https://lists.qt-project.org/pipermail/development/2020-February/039004.html).)
- the corresponding *clazy* [check](https://github.com/KDE/clazy/blob/master/docs/checks/README-qt-keywords.md) is disabled by default, quoting:  
 _Warns when using Qt keywords such as emit, slots, signals or foreach. This check is disabled by default and must be explicitly enabled, since using the above Qt keywords is fine unless you're using 3rdparty headers that also define them, in which case you'll want to use Q_EMIT, Q_SLOTS, Q_SIGNALS or Q_FOREACH instead._  
(We don't and will not use headers that define them in near future.)
- when we decide to make the change, we should do it in one big commit.

You mentioned that you read that lowercase macros will be deprecated in Qt 6, do you have a link to share regarding that?